.github/scripts/build_triton_wheel.py

```diff
@@ -36,15 +36,6 @@ def patch_setup_py(path: Path, *, version: str, name: str = "triton") -> None:
         f.write(orig)
 
 
-def patch_init_py(path: Path, *, version: str) -> None:
-    with open(path) as f:
-        orig = f.read()
-    # Replace version
-    orig = check_and_replace(orig, "__version__ = '2.0.0'", f"__version__ = \"{version}\"")
-    with open(path, "w") as f:
-        f.write(orig)
-
-
```

Comment:
Please don't delete that one, but replace hardcoded `2.0.0` with `read_triton_version()`